### ISSUE TITLE
Translate all the recent accessibility text

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#881]
We're making a lot of accessibility fixes which include instructional text, which will need to be translated. Rather than delay the accessibility fixes, they are being done in English for now, but let's use this ticket to make sure they all get translated.

**Comment by LucyGwilliamAdmin** [comment]
To be done after DAC retest.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @AnnCorp I'm proposing we get this into 1.2.0 instead of waiting for 1.3.0 - for a few reasons:

1. I've added English translations to sdg-translations
2. I've submitted [a PR to have Open SDG use those translations](https://github.com/open-sdg/open-sdg/pull/987)
3. We now have Weblate (see below)

To expand on the Weblate part, I'm wondering what you think of this as a new general workflow for adding translations:

1. Add English translations to sdg-translations (which adds them to Weblate automatically)
2. Code open-sdg to use those translations
3. Wait and see if anyone translates them into other languages in Weblate
4. When it is time for a new open-sdg release, we go into Weblate and do machine translations for anything still untranslated

This way we're guaranteed to at least have machine translations in place before new releases, but we also have the human translators some time to translate manually. FYI doing machine translations in Weblate is very easy thanks to the "Automatic suggestions" tab.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning if pretty much already there then I guess makes sense to get it into 1.2.0.

I have a quick question about Weblate - if a translations file is missing a translation completely e.g. doesn't even have the key (`[KEY]: '' `), would Weblate still pick this up a missing translation for a particular language?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Yes, Weblate sees these as untranslated. For example go to the [Spanish indicator group](https://hosted.weblate.org/projects/sdg-translations/indicator/es/) and click on "Not translated strings". It will list the new ones that I just added as English-only.

**Comment by brockfanning** [comment]
This has been merged.
